utkdigitalinitiatives/utk_isl_xml_forms
# Create undergraduate theses form and test for CRUD operations.

**JIRA Ticket**: [TRAC-1040](https://jira.lib.utk.edu/browse/TRAC-1040)

# What does this Pull Request do?

Adds a new metadata form for Undergraduate theses.

# What's new?
Adds a form based on the undergraduate theses section here:
https://docs.google.com/spreadsheets/d/1JOXRWpwz0nAnWitHmGyLbfHTTpiBXXXAeqfSeMAJhPo/edit#gid=0

# How should this be tested?

* Test the entire workflow:
    * Import the Form
    * Associate it with a content model
    * Apply any additional related transforms
    * Create a new record and select the newly associated form
    * Edit that record to verify proper CRUD behavior

# Interested parties
@CanOfBees @cdeaneGit
